onboarding.md

```diff
@@ -0,0 +1,26 @@
+# Onboarding
```

Comment:
I think we should likely call this `membership expectations` rather than onboarding... as this does not appear to include specifics related to onboarding folks to the group

Reply:
fair point